libs/ui-lib/lib/common/config/docs_links.ts

```diff
@@ -0,0 +1,83 @@
+export const getShortOpenshiftVersion = (ocpVersion?: string) =>
+  ocpVersion?.split('.').slice(0, 2).join('.');
+
+export const ASSISTED_INSTALLER_DOCUMENTATION_LINK =
+  'https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/index';
+
+export const OPENSHIFT_LIFE_CYCLE_DATES_LINK =
+  'https://access.redhat.com/support/policy/updates/openshift#dates';
+export const getOpenShiftNetworkingDocsLink = (ocpVersion?: string) =>
+  `https://docs.openshift.com/container-platform/${
+    getShortOpenshiftVersion(ocpVersion) || '4.13'
+  }/installing/installing_bare_metal/installing-bare-metal.html#installation-network-user-infra_installing-bare-metal`;
+export const CLUSTER_MANAGER_SITE_LINK = 'https://console.redhat.com/openshift/install/pull-secret';
+export const PULL_SECRET_INFO_LINK = CLUSTER_MANAGER_SITE_LINK;
+export const ODF_REQUIREMENTS_LINK =
+  'https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.15';
+export const VSPHERE_CONFIG_LINK = 'https://access.redhat.com/solutions/6677901';
```

Comment:
I think this would be a good opportunity to sort the links more.

For example, Vsphere and Nutanix should be next to each other. The operators similarly should be grouped together.

This would allow to understand better which links the UI is using.